lua/cmake-tools/utils.lua

```diff
@@ -61,6 +61,22 @@ function utils.file_exists(path)
   end
   return true
 end
+function utils.get_tmp_dir()
+  return vim.fn.stdpath("data") .. "/cmake-tools-tmp"
+end
+
+---function returnds path to the tmp_file
+---@param file_name string name of the file
+---@return string
+function utils.get_tmp_file_path(file_name)
+  return utils.get_tmp_dir() .. "/" .. file_name
+end
+
+function utils.create_tmp_file(file_name)
+  utils.mkdir(utils.get_tmp_dir())
+  local tmp_file = io.open(utils.get_tmp_file_path(file_name), "w")
+  tmp_file:close()
+end
```

Comment:
These functions are specific to `terminal`, should be moved to there.

Reply:
my idea was that the tmp directory handling is not specyfic to terminal, it is just that only terminal uses it for now, but if we ever need any other tmp file stored we could use the same functions

Reply:
I still think they are too specific to be in utils.